src/SPC/util/SPCConfigUtil.php

```diff
@@ -92,7 +92,11 @@ private function getLibsString(array $libraries): string
         }
         // patch: imagick (imagemagick wrapper) for linux needs -lgomp
         if (in_array('imagemagick', $libraries) && PHP_OS_FAMILY === 'Linux') {
-            $short_name[] = '-lgomp';
+            if (getenv('SPC_LIBC') === 'glibc') {
+                $short_name[] = '-l:libgomp.a';
+            } else {
+                $short_name[] = '-lgomp';
+            }
```

Comment:
we need to check how this affects uses of the embed SAPI. They need to also find -l:libgomp.a